docker/ghd.py

```diff
@@ -99,10 +102,22 @@ def main_group():
         if len(require_context) != 1:
             raise RuntimeError("When not requiring any context by using '-', no other contexts must be required")
         require_context = []
+        require_context_str = color_error("none")
     elif "+" in require_context:
         if len(require_context) != 1:
             raise RuntimeError("When requiring all contexts by using '+', no other contexts must be required")
         require_context = None
+        require_context_str = color_success("all")
+    else:
+        require_context_str = color_unknown(", ".join(require_context))
+
+    print_info(f"{repo}@{ref} will be deployed to {environment}")
```

Comment:
logging statt print?

Reply:
Grundsätzlich ja, ein Umbau auf Standard-Logging-Lösungen ist geplant.

Reply:
@stohrendorf supergut wäre noch zu Sicherheit irgendwas wie `git describe --tags`-mäsiges neben `{repo}@{ref}` in Klammern anzeigen, sodass man noch besser sieht, was war der letzte stand, und was deployed wird. Oder erste Zeile aus dem Commit-Message? Muss nicht hier gemacht werden, nur eine Idee, falls du schon selbst drangedacht hast und kann mit eine Zeile umgesetzt werden...